app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -82,6 +84,7 @@ define(function (require, exports, module) {
             if (passwordCheckStatus === 'BLOOMFILTER_HIT' ||
```

Comment:
OMG, great catch @vbudhram . We should be logging those as weak passwords as well!!